DogServiceExample/src/main/java/Json/JsonHelper.java

```diff
@@ -1,18 +0,0 @@
-package Json;
```

Comment:
Package names should not start with capital letter.
Also they usually start with the groupId, so `com.common.json` in your case.